migrations/20220328222746_BOOK_TABLE.go

```diff
@@ -1,3 +1,4 @@
+// nolint:dupl //Cannot use same test for different migrations
```

Comment:
Is this a lint comment? Remove if not relevant.

Reply:
this comment tells the linter to ignore a specific check on the lines that follow.